.taskcat.yml

```diff
@@ -3,53 +3,70 @@ project:
   shorten_stack_name: true
   s3_regional_buckets: true
   regions:
-    - us-east-2
+    - us-east-1
   artifact_regions:
-    - us-east-2
+    - us-east-1
 tests:
   default:
     template: ./templates/main-org.template.yaml
     regions:
-    - us-east-2
+    - us-east-1
     parameters:
-      QSS3KeyPrefix: "quickstart-sumo-logic-cloudsiem/"
+      QSS3KeyPrefix: "cfn-ps-sumo-logic-cloudsiem/"
       QSS3BucketName:  "$[taskcat_autobucket_prefix]"
-      QSS3BucketRegion: "us-east-2"
-      # Section1SumoLogicDeployment: "us1"
-      # Section1SumoLogicAccessID: ""
-      # Section1SumoLogicAccessKey: ""
-      # Section1SumoLogicOrganizationId: ""
+      QSS3BucketRegion: "us-east-1"
+      QSVersion: "2"
+      Section1SumoLogicDeployment: "us2"
+      Section1SumoLogicAccessID: $[taskcat_ssm_/sumologic/Section1SumoLogicAccessID]
+      Section1SumoLogicAccessKey: $[taskcat_ssm_/sumologic/Section1SumoLogicAccessKey]
+      Section1SumoLogicOrganizationId: $[taskcat_ssm_/sumologic/Section1SumoLogicOrganizationId]
       Section1SecurityToolingAccountId: ""
```

Comment:
@langmtt please change this to the below so we can execute the automated tests to merge:
Section1SecurityToolingAccountId: $[taskcat_ssm_/sumologic/Section1SecurityToolingAccountId]